soccer/src/soccer/strategy/agent/position/defense.cpp

```diff
@@ -4,8 +4,16 @@ namespace strategy {
 
 Defense::Defense(int r_id) : Position(r_id, "Defense"), marker_{field_dimensions_} {}
 
+Defense::Defense(const Position& other) : Position{other}, marker_{field_dimensions_} {
+    SPDLOG_INFO("HELLO {}", robot_id_);
```

Comment:
Does the SPDLOG command need to be here? If so, possibly make it more descriptive.